lms/templates/ccx/schedule.html

```diff
@@ -124,5 +124,18 @@
         'dateFormat': 'yy-mm-dd',
         'autoclose': true
     });
+    $('.datepair .date').change(function() {
```

Comment:
@carsongee now using date picker api to grab date, the aim of this PR is to disallow date years like 0001,0002.... Currently date pickers allows date format like 0001-2-23 but javascrip and date picker assume it as 2001-2-23, So text field shows date  0001-2-23  but date picker dialog show it as 2001-2-23. It is ambiguous. 
in this PR I made date consistent i.e same on both date picker dialog and text field. When use adds date in text field like 0001-2-23 app convert it to 2001-2-23. A valid year.